src/components/input/range/index.d.ts

```diff
@@ -0,0 +1,19 @@
+import { ChangeEvent, FC, SyntheticEvent } from "react"
+
+export interface RangeInputProps {
+  "data-testid"?: string
+  max?: number
+  min?: number
+  onChange: (e: ChangeEvent<HTMLInputElement>) => void
+  onClick?: (e: SyntheticEvent<HTMLButtonElement>) => void
+  onInput?: (e: SyntheticEvent<HTMLButtonElement>) => void
+  step?: number
+  value: number
+  [key: string]: any
```

Comment:
What do what you like to achieve here? By adding this property it means that whatever property you pass to the `InputRange` it will be valid, e.x if i pass the property `randomPorperty = "adads"` typescript will not warn me that i have passed a property that does not exist in the types.

Reply:
It's a pattern that all components of our library have!
That's why I have also included it here! 😄